packages/victory-core/src/victory-primitives/background.js

```diff
@@ -4,58 +4,33 @@ import CommonProps from "../victory-util/common-props";
 import Rect from "./rect";
 import Circle from "./circle";
 
-const getRangeBounds = (props) => ({ x: props.range.x[0], y: props.range.y[1] });
-const getHeight = (props) => (props.polar ? props.range.y[1] : props.range.y[0] - props.range.y[1]);
-const getWidth = (props) => props.range.x[1] - props.range.x[0];
-
 const Background = (props) => {
   return props.polar
     ? React.cloneElement(props.circleComponent, {
         ...props.events,
-        style: props.style && props.style.background,
+        style: props.style,
         role: props.role,
         shapeRendering: props.shapeRendering,
-        cx: "50%",
-        cy: "50%",
-        r: getHeight(props)
+        cx: origin.x,
+        cy: origin.y,
+        r: props.height
       })
     : React.cloneElement(props.rectComponent, {
         ...props.events,
-        style: props.style && props.style.background,
+        style: props.style,
         role: props.role,
         shapeRendering: props.shapeRendering,
-        x: getRangeBounds(props).x,
-        y: getRangeBounds(props).y,
-        width: getWidth(props),
-        height: getHeight(props)
+        x: props.scale.x.range(),
```

Comment:
I think range is a tuple, rather than a single value, so this wont work. Also, I'd like to move this calculation to  `getBackgroundWithProps`

Reply:
Yep, we got that. sorry, this is *very* WIP... should've moved to draft! We just pushed so we could both have it locally before lunch